node/packages/parser/src/ethereum/utils.ts

```diff
@@ -27,9 +27,12 @@ export const txInteractsWithContract = (tx: Tx, contract: string) => {
   return receiveAddress === contract
 }
 
-export const getYearnTokenVaultAddresses = async (): Promise<Array<string> | undefined> => {
-  const yearnTokenVaultResponse = await axios.get<YearnTokenVault[]>('https://api.yearn.finance/v1/chains/1/vaults/all')
-  return yearnTokenVaultResponse.status === 200 ? yearnTokenVaultResponse.data.map((vault) => vault.address) : undefined
+export const getYearnTokenVaultAddresses = async (provider: ethers.providers.JsonRpcProvider) => {
+  const network = 1 // 1 for mainnet
```

Comment:
The eth parser is initialized with a network that we should pass to the yearn parse for use in initializing the yearnSdk. Does it support testnet? We can switch on network to convert to yearn network value (do they have an enum we can use instead of the int?) and handle testnet as needed depending on support.

Reply:
The Yearn SDK doesn't support testnet/ropsten - the only overlap we have is with `mainnet`, though they don't export an enumerated, named type from the SDK.

I've improved clarity around this in https://github.com/shapeshift/unchained/pull/284/commits/9d451a4291cebcab62365fd6a9da591b59f22c03, which may be as good as we get for now.